src/main/java/com/blogify/blogapi/repository/model/PostReaction.java

```diff
@@ -0,0 +1,24 @@
+package com.blogify.blogapi.repository.model;
+
+import com.blogify.blogapi.utils.ReactionType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
```

Comment:
L3 - avoid importation with '.*'
for example: 
import javax.persistence.*;
divide it into multiple importations and configure IntelliJ to not do it automatically